juriscraper/lib/string_utils.py

```diff
@@ -434,7 +434,10 @@ def trunc(s, length, ellipsis=None):
 
 
 def convert_date_string(date_string):
-    """Convert date string into standard date object"""
+    """Sanitize date string and convert into standard date object"""
```

Comment:
I usually chain these when I have a bunch of strips and replaces to run, but only a style comment.